README.md

```diff
@@ -11,9 +7,10 @@ This repository is no longer actively maintained and is deprecated.
 
 A TypeScript/JavaScript library for inserting and extracting IPFS data to and from Tableland queries using the `@tableland/sdk`.
 
+This library is only compartible with version 3 of `@tableland/sdk`.
```

Comment:
Looks like a spelling error, should it be "compatible"?

Reply:
Fixed.